src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1307,7 +1307,8 @@ void fromNetwork (
     // Send request
 
     const int RPC_REPLY_MAX_BYTES (256*1024*1024);
-    const int RPC_NOTIFY_SECONDS (600);
+    using namespace std::chrono_literals;
+    auto constexpr RPC_NOTIFY_SECONDS = 10min;
```

Comment:
Here's another one that could use a rename. No need for `_SECONDS` when the value is minutes.